### PR TITLE
feat: adoption guide update by adding more pages and fixes

### PR DESCRIPTION
**Pull request by bdryanovski**
* fixing broken snippets
 * making code example in side-by-side mode
 * adding pages based on the eslint list
 * adding not-ready component to mark the pages (needs more work)

### CODE REVIEW

**Review by bbogdanov — APPROVED** [review]
`feat(adoption): adding pages for all available components`

The fixes are not mandatory so I guess we can skip them in the title

File: apps/adoption/src/app/pages/getting-started.page.ts [hunk 1]
```diff
@@ -10,8 +10,42 @@ import { SupportedTemplates } from '../templates';
 
 @Component({
   selector: 'app-home',
-  templateUrl: './getting-started.page.html',
   providers: [StackblitzService],
+  template: `
+    <h1>Getting Started</h1>
+
+    <h3>For who is this guide?</h3>
```
Comment (Jinnie) [inline apps/adoption/src/app/pages/getting-started.page.ts]: Who is this guide for

File: apps/adoption/src/app/pages/getting-started.page.ts [hunk 2]
```diff
@@ -10,8 +10,42 @@ import { SupportedTemplates } from '../templates';
...
+    <p>
+      This guide is for everyone that want and needs to adopt our new Core components into there existing Clarity
```
Comment (Jinnie) [inline apps/adoption/src/app/pages/getting-started.page.ts:19]: ...that wants and needs....  ... into their existing....

File: apps/adoption/src/app/pages/getting-started.page.ts [hunk 3]
```diff
@@ -10,8 +10,42 @@ import { SupportedTemplates } from '../templates';
...
+      Angular application. It's design to help you rework and adopt in the best way and still get the same result as
```
Comment (Jinnie) [inline apps/adoption/src/app/pages/getting-started.page.ts]: It's designed to.....

File: apps/adoption/src/app/pages/getting-started.page.ts [hunk 4]
```diff
@@ -10,8 +10,42 @@ import { SupportedTemplates } from '../templates';
...
+      before.
+    </p>
+
+    <h3>How to use this guide?</h3>
+    <p>
+      The guide is structure into sections, every section will provide examples based on our Angular components and
```
Comment (Jinnie) [inline apps/adoption/src/app/pages/getting-started.page.ts]: ...is structured...

File: apps/adoption/src/app/pages/getting-started.page.ts [hunk 5]
```diff
@@ -10,8 +10,42 @@ import { SupportedTemplates } from '../templates';
...
+      there Core counterparts. Into the form of guides, snippets and corner cases that will help you with the adoption
```
Comment (Jinnie) [inline apps/adoption/src/app/pages/getting-started.page.ts]: ... their Core counterparts.

**Review by Jinnie — CHANGES_REQUESTED** [review]
Good progress!

Only a couple of typo fixes from me in this review.

**Review by Jinnie — APPROVED** [review]
Thank you!

**Comment by bbogdanov** [comment]
I know it might get tedious and I am sorry about it. 

`feat(adoption): createing pages structure`  -> `feat(adoption): creating pages structure` 

The additional `e` is the change.

**Pull request merged**